lumibot/tools/indicators.py

```diff
@@ -713,6 +714,9 @@ def create_tearsheet(
         logging.error("Not enough data to create a tearsheet, at least 2 days of data are required. Skipping")
         return
 
+    # Set the name of the benchmark column so that quantstats can use it in the report
+    df_final["benchmark"].name = str(benchmark_asset)
+
```

Comment:
Seems to me that the "title" would be a nicer place to put the benchmark name, but I'm having a hard time visualizing what the final output looks like.

Reply:
this is just the way that the quant stats library works, unfortunately we dont have a choice here